src/reference-assemblies/build.proj

```diff
@@ -1,15 +1,15 @@
 <Project>
+  <Import Project="../../dir.props" />
```

Comment:
A better way to do this is to say:

`<Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />`

Which tells MSBuild to find the first `dir.props` file in this directory, or any directory above this directory.  This allows for a new `dir.props` file to be added in the directory directly above this file, and this code doesn't need to change - it automatically gets picked up.

Reply:
I saw @weshaggard added in his restructuring PR.  I'll add it here.